### PR TITLE
feat: added CLI tool to easily get started using the project

### LINKED ISSUE #33: Simplify the getting started process
Currently, we need to read too many texts in order to get started. This is not acceptable as the goal of this project is to make the process stupid simple. An interactive CLI could be a great idea.

### PR DESCRIPTION
**Pull request by yann510**
closes #33 

Please checkout the branch and test this out.
Be careful, the final step will rename the whole project...

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,7 +37,8 @@
     "rename-project": "node ./tools/rename-project.js --project-name",
     "remap-api-url": "node ./.do/remap-api-url.js",
     "remap-redoc": "node ./.do/remap-redoc.js",
-    "commit": "cz"
+    "commit": "cz",
+    "get-started": "npm i --prefix tools/getting-started && npm run start --prefix tools/getting-started"
```
Comment (GabrielBottari) [inline package.json:41]: why not use npx?

File: tools/getting-started/src/error.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { Text } from "ink";
+import React from "react"
+
+interface Props {
```
Comment (GabrielBottari) [inline tools/getting-started/src/error.tsx:4]: we use 2 space padding everywhere in the project. make sure the new files are consistent with this

File: tools/getting-started/src/label-value-input.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+import TextInput from "ink-text-input"
+import React from "react"
+
+import Error from "./error"
+import IsValidIcon from "./is-valid-icon"
+import { InputValue } from "./ui"
+
+interface Props {
+	label: string
+  placeholder?: string
```
Comment (GabrielBottari) [inline tools/getting-started/src/label-value-input.tsx:11]: why does this attribute has a different padding

**Review by GabrielBottari — CHANGES_REQUESTED** [review]
I'm also expecting some detailed usage instructions in the readme

File: package.json (hunk 1 above)
Comment (yann510) [reply package.json:41]: We need the deps installed locally.

File: tools/getting-started/src/error.tsx (hunk 2 above)
Comment (yann510) [reply tools/getting-started/src/error.tsx:4]: I'm a bit confused with your comment as all my files use 2 spaces...

**Review by GabrielBottari — APPROVED** [review]

**Pull request merged**